src/marketplace/Apps.Service/Controllers/AppReleaseProcessController.cs

```diff
@@ -78,7 +78,7 @@ public async Task<NoContentResult> UpdateApp([FromRoute] Guid appId, [FromBody]
     }
 
     /// <summary>
-    /// Upload document for active apps in the marketplace for given appId for same company as user
+    /// Upload document for  apps in status CREATED and document in status PENDING in the marketplace for given appId for same company as user
```

Comment:
there is a double whitespace between `for` and `apps`
```suggestion
    /// Upload document for apps in status CREATED and document in status PENDING in the marketplace for given appId for same company as user
```

Reply:
fixed